devices.js

```diff
@@ -868,7 +868,7 @@ const devices = [
             fz.ignore_occupancy_report,
             fz.ignore_illuminance_report, fz.ignore_time_read,
         ],
-        toZigbee: [tz.on_off, tz.xiaomi_switch_power_outage_memory],
+        toZigbee: [tz.on_off, tz.xiaomi_power, tz.xiaomi_switch_power_outage_memory],
```

Comment:
Please also add tz.xiaomi_power to the other devices having fz.xiaomi_power (like I did in https://github.com/Koenkk/zigbee-herdsman-converters/pull/1497/files)

Reply:
Done